### ISSUE TITLE
Video Player "UnimplementedError: init() has not been implemented."

### ISSUE THREAD

**Issue by Mitchnoff** [flutter/flutter#122738]
## Steps to Reproduce

I am attempting to recreate the instagram UI using Chewie and Video Player from [this video](https://www.youtube.com/watch?v=uF0dAMDbenQ&t=177s). The code for this video can be found from [this repo](https://github.com/pktintali/flutter_shorts/blob/main/lib/screens/content_screen.dartl)

1. Execute `flutter run` on the code sample 
2. Error occurs on launch given the simple program.

**Expected results:** 

I expect to be able to get a video from an online resource and play it using the video player.

**Actual results:** 

When running, there is a perpetual loading screen on the reels screen. I believe this is due to the exception being thrown and never actually loading the video. From my research I have found very few results similar to this.

I tried downgrading video_player from 2.6.0 to 2.5.3 to see if the newer version of it caused the problem, but the issue persisted.

<details>
  <summary>Logs</summary>

```
[  +91 ms] executing: [C:\src\flutter/] git -c log.showSignature=false log -n 1 --pretty=format:%H
[ +153 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[        ] 135454af32477f815a7525073027a3ff9eff1bfd
[        ] executing: [C:\src\flutter/] git tag --points-at 135454af32477f815a7525073027a3ff9eff1bfd
[ +192 ms] Exit code 0 from: git tag --points-at 135454af32477f815a7525073027a3ff9eff1bfd
[        ] 3.3.10
[  +69 ms] executing: [C:\src\flutter/] git rev-parse --abbrev-ref --symbolic @{upstream}
[  +84 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{upstream}
[        ] origin/stable
[        ] executing: [C:\src\flutter/] git ls-remote --get-url origin
[  +84 ms] Exit code 0 from: git ls-remote --get-url origin
[        ] https://github.com/flutter/flutter.git
[ +137 ms] executing: [C:\src\flutter/] git rev-parse --abbrev-ref HEAD
[ +191 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[        ] stable
[ +139 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +4 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[  +70 ms] executing: C:\Users\allenm\AppData\Local\Android\sdk\platform-tools\adb.exe devices -l
[  +91 ms] List of devices attached
[   +7 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[   +6 ms] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[  +82 ms] Multiple devices found:
[  +51 ms] Windows (desktop) ΓÇó windows ΓÇó windows-x64    ΓÇó Microsoft Windows [Version 10.0.19044.2604]
[        ] Chrome (web)      ΓÇó chrome  ΓÇó web-javascript ΓÇó Google Chrome 109.0.5414.120
[        ] Edge (web)        ΓÇó edge    ΓÇó web-javascript ΓÇó Microsoft Edge 110.0.1587.63
[        ] [1]: Windows (windows)
[        ] [2]: Chrome (chrome)
[        ] [3]: Edge (edge)
[   +1 ms] Please choose one (To quit, press "q/Q")
[        ] : 
[ +171 ms] 1
[  +27 ms] Skipping pub get: version match.
[  +58 ms] Found plugin path_provider_linux at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\path_provider_linux-2.1.9\
[   +6 ms] Found plugin path_provider_windows at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\path_provider_windows-2.1.4\
[   +7 ms] Found plugin shared_preferences at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences-2.0.18\
[   +1 ms] Found plugin shared_preferences_android at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_android-2.0.16\
[   +2 ms] Found plugin shared_preferences_foundation at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_foundation-2.1.4\
[   +3 ms] Found plugin shared_preferences_linux at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_linux-2.1.4\
[   +3 ms] Found plugin shared_preferences_web at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_web-2.0.5\
[   +1 ms] Found plugin shared_preferences_windows at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_windows-2.1.4\
[   +7 ms] Found plugin video_player at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\video_player-2.6.0\
[   +1 ms] Found plugin video_player_android at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\video_player_android-2.4.2\
[   +1 ms] Found plugin video_player_avfoundation at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\video_player_avfoundation-2.4.1\
[   +2 ms] Found plugin video_player_web at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\video_player_web-2.0.15\
[   +1 ms] Found plugin wakelock at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\wakelock-0.6.2\
[   +1 ms] Found plugin wakelock_macos at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\wakelock_macos-0.4.0\
[   +5 ms] Found plugin wakelock_web at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\wakelock_web-0.4.0\
[  +83 ms] Found plugin path_provider_linux at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\path_provider_linux-2.1.9\
[   +2 ms] Found plugin path_provider_windows at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\path_provider_windows-2.1.4\
[   +3 ms] Found plugin shared_preferences at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences-2.0.18\
[        ] Found plugin shared_preferences_android at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_android-2.0.16\
[        ] Found plugin shared_preferences_foundation at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_foundation-2.1.4\
[        ] Found plugin shared_preferences_linux at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_linux-2.1.4\
[   +1 ms] Found plugin shared_preferences_web at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_web-2.0.5\
[        ] Found plugin shared_preferences_windows at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_windows-2.1.4\
[   +4 ms] Found plugin video_player at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\video_player-2.6.0\
[   +1 ms] Found plugin video_player_android at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\video_player_android-2.4.2\
[   +1 ms] Found plugin video_player_avfoundation at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\video_player_avfoundation-2.4.1\
[   +4 ms] Found plugin video_player_web at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\video_player_web-2.0.15\
[   +2 ms] Found plugin wakelock at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\wakelock-0.6.2\
[   +1 ms] Found plugin wakelock_macos at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\wakelock_macos-0.4.0\
[   +2 ms] Found plugin wakelock_web at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\wakelock_web-0.4.0\
[  +68 ms] Found plugin path_provider_linux at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\path_provider_linux-2.1.9\
[   +1 ms] Found plugin path_provider_windows at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\path_provider_windows-2.1.4\
[   +4 ms] Found plugin shared_preferences at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences-2.0.18\
[        ] Found plugin shared_preferences_android at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_android-2.0.16\
[        ] Found plugin shared_preferences_foundation at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_foundation-2.1.4\
[        ] Found plugin shared_preferences_linux at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_linux-2.1.4\
[   +1 ms] Found plugin shared_preferences_web at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_web-2.0.5\
[        ] Found plugin shared_preferences_windows at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_windows-2.1.4\
[   +3 ms] Found plugin video_player at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\video_player-2.6.0\
[        ] Found plugin video_player_android at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\video_player_android-2.4.2\
[        ] Found plugin video_player_avfoundation at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\video_player_avfoundation-2.4.1\
[   +1 ms] Found plugin video_player_web at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\video_player_web-2.0.15\
[        ] Found plugin wakelock at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\wakelock-0.6.2\
[        ] Found plugin wakelock_macos at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\wakelock_macos-0.4.0\
[   +1 ms] Found plugin wakelock_web at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\wakelock_web-0.4.0\
[  +58 ms] Generating C:\Users\allenm\Desktop\work\artc\artc-frontend\artc_frontend\android\app\src\main\java\io\flutter\plugins\GeneratedPluginRegistrant.java
[ +136 ms] Initializing file store
[  +17 ms] Skipping target: gen_localizations
[   +6 ms] gen_dart_plugin_registrant: Starting due to {InvalidatedReasonKind.inputChanged: The following inputs have updated contents: C:\Users\allenm\Desktop\work\artc\artc-frontend\artc_frontend\.dart_tool\package_config_subset,C:\Users\allenm\Desktop\work\artc\artc-frontend\artc_frontend\.dart_tool\flutter_build\dart_plugin_registrant.dart}
[  +33 ms] Found plugin path_provider_linux at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\path_provider_linux-2.1.9\
[   +2 ms] Found plugin path_provider_windows at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\path_provider_windows-2.1.4\
[   +6 ms] Found plugin shared_preferences at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences-2.0.18\
[   +1 ms] Found plugin shared_preferences_android at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_android-2.0.16\
[   +1 ms] Found plugin shared_preferences_foundation at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_foundation-2.1.4\
[   +1 ms] Found plugin shared_preferences_linux at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_linux-2.1.4\
[   +2 ms] Found plugin shared_preferences_web at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_web-2.0.5\
[   +2 ms] Found plugin shared_preferences_windows at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_windows-2.1.4\
[   +5 ms] Found plugin video_player at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\video_player-2.6.0\
[   +1 ms] Found plugin video_player_android at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\video_player_android-2.4.2\
[   +2 ms] Found plugin video_player_avfoundation at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\video_player_avfoundation-2.4.1\
[   +1 ms] Found plugin video_player_web at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\video_player_web-2.0.15\
[   +1 ms] Found plugin wakelock at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\wakelock-0.6.2\
[   +1 ms] Found plugin wakelock_macos at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\wakelock_macos-0.4.0\
[   +3 ms] Found plugin wakelock_web at C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\wakelock_web-0.4.0\
[  +13 ms] gen_dart_plugin_registrant: Complete
[   +3 ms] Skipping target: _composite
[   +2 ms] complete
[   +5 ms] Launching lib\main.dart on Windows in debug mode...
[   +7 ms] C:\src\flutter\bin\cache\dart-sdk\bin\dart.exe --disable-dart-dev C:\src\flutter\bin\cache\dart-sdk\bin\snapshots\frontend_server.dart.snapshot --sdk-root C:\src\flutter\bin\cache\artifacts\engine\common\flutter_patched_sdk/ --incremental --target=flutter --debugger-module-names --experimental-emit-debug-metadata -DFLUTTER_WEB_AUTO_DETECT=true --output-dill C:\Users\allenm\AppData\Local\Temp\flutter_tools.bba22bd4\flutter_tool.f871c035\app.dill --packages C:\Users\allenm\Desktop\work\artc\artc-frontend\artc_frontend\.dart_tool\package_config.json -Ddart.vm.profile=false -Ddart.vm.product=false --enable-asserts --track-widget-creation --filesystem-scheme org-dartlang-root --initialize-from-dill build\c075001b96339384a97db4862b8ab8db.cache.dill.track.dill --source C:\Users\allenm\Desktop\work\artc\artc-frontend\artc_frontend\.dart_tool\flutter_build\dart_plugin_registrant.dart --source package:flutter/src/dart_plugin_registrant.dart -Dflutter.dart_plugin_registrant=file:///C:/Users/allenm/Desktop/work/artc/artc-frontend/artc_frontend/.dart_tool/flutter_build/dart_plugin_registrant.dart --verbosity=error --enable-experiment=alternative-invalidation-strategy
[  +30 ms] executing: C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere.exe -format json -products * -utf8 -latest -version 16 -requires Microsoft.VisualStudio.Workload.NativeDesktop Microsoft.VisualStudio.Component.VC.Tools.x86.x64 Microsoft.VisualStudio.Component.VC.CMake.Project
[ +123 ms] Exit code 0 from: C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere.exe -format json -products * -utf8 -latest -version 16 -requires Microsoft.VisualStudio.Workload.NativeDesktop Microsoft.VisualStudio.Component.VC.Tools.x86.x64 Microsoft.VisualStudio.Component.VC.CMake.Project
[        ] [
             {
               "instanceId": "f4ab6491",
               "installDate": "2023-01-13T18:19:43Z",
               "installationName": "VisualStudio/17.4.4+33213.308",
               "installationPath": "C:\\Program Files\\Microsoft Visual Studio\\2022\\Community",
               "installationVersion": "17.4.33213.308",
               "productId": "Microsoft.VisualStudio.Product.Community",
               "productPath": "C:\\Program Files\\Microsoft Visual Studio\\2022\\Community\\Common7\\IDE\\devenv.exe",
               "state": 4294967295,
               "isComplete": true,
               "isLaunchable": true,
               "isPrerelease": false,
               "isRebootRequired": false,
               "displayName": "Visual Studio Community 2022",
               "description": "Powerful IDE, free for students, open-source contributors, and individuals",
               "channelId": "VisualStudio.17.Release",
               "channelUri": "https://aka.ms/vs/17/release/channel",
               "enginePath": "C:\\Program Files (x86)\\Microsoft Visual Studio\\Installer\\resources\\app\\ServiceHub\\Services\\Microsoft.VisualStudio.Setup.Service",
               "installedChannelId": "VisualStudio.17.Release",
               "installedChannelUri": "https://aka.ms/vs/17/release/channel",
               "releaseNotes": "https://docs.microsoft.com/en-us/visualstudio/releases/2022/release-notes-v17.4#17.4.4",
               "thirdPartyNotices": "https://go.microsoft.com/fwlink/?LinkId=661288",
               "updateDate": "2023-01-13T18:19:43.1168229Z",
               "catalog": {
                 "buildBranch": "d17.4",
                 "buildVersion": "17.4.33213.308",
                 "id": "VisualStudio/17.4.4+33213.308",
                 "localBuild": "build-lab",
                 "manifestName": "VisualStudio",
                 "manifestType": "installer",
                 "productDisplayVersion": "17.4.4",
                 "productLine": "Dev17",
                 "productLineVersion": "2022",
                 "productMilestone": "RTW",
                 "productMilestoneIsPreRelease": "False",
                 "productName": "Visual Studio",
                 "productPatchVersion": "4",
                 "productPreReleaseMilestoneSuffix": "1.0",
                 "productSemanticVersion": "17.4.4+33213.308",
                 "requiredEngineVersion": "3.4.2246.31370"
               },
               "properties": {
                 "campaignId": "2030:fdf732342b2846c0966e2539c1034897",
                 "canceled": "0",
                 "channelManifestId": "VisualStudio.17.Release/17.4.4+33213.308",
                 "nickname": "",
                 "setupEngineFilePath": "C:\\Program Files (x86)\\Microsoft Visual Studio\\Installer\\setup.exe"
               }
             }
           ]
[   +4 ms] Building Windows application...
[  +23 ms] <- compile package:artc_frontend/main.dart
[   +2 ms] executing: C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe -S C:\Users\allenm\Desktop\work\artc\artc-frontend\artc_frontend\windows -B build\windows -G Visual Studio 17 2022
[ +159 ms] -- Selecting Windows SDK version 10.0.22000.0 to target Windows 10.0.19044.
[  +79 ms] -- Configuring done
[ +196 ms] -- Generating done
[  +11 ms] -- Build files have been written to: C:/Users/allenm/Desktop/work/artc/artc-frontend/artc_frontend/build/windows
[   +9 ms] executing: C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe --build build\windows --config Debug --target INSTALL --verbose
[ +278 ms] MSBuild version 17.4.1+9a89d02ff for .NET Framework
[  +89 ms] Build started 3/15/2023 1:21:05 PM.
[ +381 ms] Project "C:\Users\allenm\Desktop\work\artc\artc-frontend\artc_frontend\build\windows\INSTALL.vcxproj" on node 1 (default targets).
[        ] Project "C:\Users\allenm\Desktop\work\artc\artc-frontend\artc_frontend\build\windows\INSTALL.vcxproj" (1) is building "C:\Users\allenm\Desktop\work\artc\artc-frontend\artc_frontend\build\windows\ZERO_CHECK.vcxproj" (2) on node 1 (default targets).
[        ] InitializeBuildStatus:
[        ]   Creating "x64\Debug\ZERO_CHECK\ZERO_CHECK.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[  +35 ms] CustomBuild:
[        ]   All outputs are up-to-date.
[  +45 ms] FinalizeBuildStatus:
[        ]   Deleting file "x64\Debug\ZERO_CHECK\ZERO_CHECK.tlog\unsuccessfulbuild".
[        ]   Touching "x64\Debug\ZERO_CHECK\ZERO_CHECK.tlog\ZERO_CHECK.lastbuildstate".
[   +1 ms] Done Building Project "C:\Users\allenm\Desktop\work\artc\artc-frontend\artc_frontend\build\windows\ZERO_CHECK.vcxproj" (default targets).
[ +189 ms] Project "C:\Users\allenm\Desktop\work\artc\artc-frontend\artc_frontend\build\windows\INSTALL.vcxproj" (1) is building "C:\Users\allenm\Desktop\work\artc\artc-frontend\artc_frontend\build\windows\ALL_BUILD.vcxproj" (3) on node 1 (default targets).
[        ] Project "C:\Users\allenm\Desktop\work\artc\artc-frontend\artc_frontend\build\windows\ALL_BUILD.vcxproj" (3) is building "C:\Users\allenm\Desktop\work\artc\artc-frontend\artc_frontend\build\windows\runner\artc_frontend.vcxproj" (4) on node 1 (default targets).
[        ] Project "C:\Users\allenm\Desktop\work\artc\artc-frontend\artc_frontend\build\windows\runner\artc_frontend.vcxproj" (4) is building "C:\Users\allenm\Desktop\work\artc\artc-frontend\artc_frontend\build\windows\flutter\flutter_assemble.vcxproj" (5) on node 1 (default targets).
[        ] InitializeBuildStatus:
[        ]   Creating "x64\Debug\flutter_assemble\flutter_assemble.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[  +11 ms] CustomBuild:
[        ]   "The build of 'C:\Users\allenm\Desktop\work\artc\artc-frontend\artc_frontend\build\windows\CMakeFiles\c70175192868fd8e1624b37e182600d5\flutter_assemble.rule' depends on 'C:\USERS\ALLENM\DESKTOP\WORK\ARTC\ARTC-FRONTEND\ARTC_FRONTEND\WINDOWS\FLUTTER\EPHEMERAL\FLUTTER_WINDOWS.DLL' which is produced by the build of 'C:\Users\allenm\Desktop\work\artc\artc-frontend\artc_frontend\build\windows\CMakeFiles\5ab9300c333e81e130ad391294421622\flutter_windows.dll.rule'. The items cannot be built in parallel."
[ +187 ms]   Generating C:/Users/allenm/Desktop/work/artc/artc-frontend/artc_frontend/windows/flutter/ephemeral/flutter_windows.dll, C:/Users/allenm/Desktop/work/artc/artc-frontend/artc_frontend/windows/flutter/ephemeral/flutter_export.h, C:/Users/allenm/Desktop/work/artc/artc-frontend/artc_frontend/windows/flutter/ephemeral/flutter_windows.h, C:/Users/allenm/Desktop/work/artc/artc-frontend/artc_frontend/windows/flutter/ephemeral/flutter_messenger.h, C:/Users/allenm/Desktop/work/artc/artc-frontend/artc_frontend/windows/flutter/ephemeral/flutter_plugin_registrar.h, C:/Users/allenm/Desktop/work/artc/artc-frontend/artc_frontend/windows/flutter/ephemeral/flutter_texture_registrar.h, C:/Users/allenm/Desktop/work/artc/artc-frontend/artc_frontend/windows/flutter/ephemeral/cpp_client_wrapper/core_implementations.cc, C:/Users/allenm/Desktop/work/artc/artc-frontend/artc_frontend/windows/flutter/ephemeral/cpp_client_wrapper/standard_codec.cc, C:/Users/allenm/Desktop/work/artc/artc-frontend/artc_frontend/windows/flutter/ephemeral/cpp_client_wrapper/plugin_registrar.cc, C:/Users/allenm/Desktop/work/artc/artc-frontend/artc_frontend/windows/flutter/ephemeral/cpp_client_wrapper/flutter_engine.cc, C:/Users/allenm/Desktop/work/artc/artc-frontend/artc_frontend/windows/flutter/ephemeral/cpp_client_wrapper/flutter_view_controller.cc, _phony_
[+2730 ms]   [  +85 ms] executing: [C:\src\flutter/] git -c log.showSignature=false log -n 1 --pretty=format:%H
[ +117 ms]   [ +148 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[        ]   [   +1 ms] 135454af32477f815a7525073027a3ff9eff1bfd
[        ]   [   +1 ms] executing: [C:\src\flutter/] git tag --points-at 135454af32477f815a7525073027a3ff9eff1bfd
[ +141 ms]   [ +140 ms] Exit code 0 from: git tag --points-at 135454af32477f815a7525073027a3ff9eff1bfd
[        ]   [        ] 3.3.10
[  +77 ms]   [  +78 ms] executing: [C:\src\flutter/] git rev-parse --abbrev-ref HEAD
[ +105 ms]   [ +104 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[        ]   [        ] stable
[ +111 ms]   [ +111 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +4 ms]   [   +4 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[ +138 ms]   [ +139 ms] executing: [C:\src\flutter/] git rev-parse --abbrev-ref --symbolic @{upstream}
[  +83 ms]   [  +83 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{upstream}
[        ]   [        ] origin/stable
[        ]   [        ] executing: [C:\src\flutter/] git ls-remote --get-url origin
[  +74 ms]   [  +74 ms] Exit code 0 from: git ls-remote --get-url origin
[        ]   [        ] https://github.com/flutter/flutter.git
[   +1 ms]   [   +1 ms] Artifact Instance of 'MaterialFonts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'GradleWrapper' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterSdk' is not required, skipping update.
[   +2 ms]   [   +2 ms] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FontSubsetArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'PubDependencies' is not required, skipping update.
[  +42 ms]   [  +42 ms] Initializing file store
[  +19 ms]   [  +18 ms] Done initializing file store
[  +70 ms]   [  +70 ms] Skipping target: gen_localizations
[  +20 ms]   [  +20 ms] Skipping target: gen_dart_plugin_registrant
[+3859 ms]   [+3859 ms] Skipping target: unpack_windows
[+1127 ms]   [+1127 ms] Skipping target: kernel_snapshot
[ +381 ms]   [ +380 ms] Skipping target: debug_bundle_windows_assets
[        ]   [        ] Persisting file store
[  +13 ms]   [  +13 ms] Done persisting file store
[   +8 ms]   [   +8 ms] build succeeded.
[  +11 ms]   [  +11 ms] "flutter assemble" took 5,881ms.
[ +207 ms]   [ +206 ms] ensureAnalyticsSent: 202ms
[        ]   [   +1 ms] Running shutdown hooks
[        ]   [        ] Shutdown hooks complete
[        ]   [        ] exiting with code 0
[ +106 ms] FinalizeBuildStatus:
[        ]   Deleting file "x64\Debug\flutter_assemble\flutter_assemble.tlog\unsuccessfulbuild".
[        ]   Touching "x64\Debug\flutter_assemble\flutter_assemble.tlog\flutter_assemble.lastbuildstate".
[        ] Done Building Project "C:\Users\allenm\Desktop\work\artc\artc-frontend\artc_frontend\build\windows\flutter\flutter_assemble.vcxproj" (default targets).
[  +31 ms] Project "C:\Users\allenm\Desktop\work\artc\artc-frontend\artc_frontend\build\windows\runner\artc_frontend.vcxproj" (4) is building "C:\Users\allenm\Desktop\work\artc\artc-frontend\artc_frontend\build\windows\flutter\flutter_wrapper_app.vcxproj" (6) on node 1 (default targets).
[        ] InitializeBuildStatus:
[        ]   Creating "flutter_wrapper_app.dir\Debug\flutter_.2C578FBB.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[   +2 ms] CustomBuild:
[        ]   All outputs are up-to-date.
[  +60 ms] ClCompile:
[        ]   All outputs are up-to-date.
[  +28 ms] Lib:
[        ]   All outputs are up-to-date.
[   +1 ms]   flutter_wrapper_app.vcxproj -> C:\Users\allenm\Desktop\work\artc\artc-frontend\artc_frontend\build\windows\flutter\Debug\flutter_wrapper_app.lib
[  +15 ms] FinalizeBuildStatus:
[        ]   Deleting file "flutter_wrapper_app.dir\Debug\flutter_.2C578FBB.tlog\unsuccessfulbuild".
[        ]   Touching "flutter_wrapper_app.dir\Debug\flutter_.2C578FBB.tlog\flutter_wrapper_app.lastbuildstate".
[   +1 ms] Done Building Project "C:\Users\allenm\Desktop\work\artc\artc-frontend\artc_frontend\build\windows\flutter\flutter_wrapper_app.vcxproj" (default targets).
[   +6 ms] InitializeBuildStatus:
[        ]   Creating "artc_frontend.dir\Debug\artc_frontend.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[   +2 ms] CustomBuild:
[        ]   All outputs are up-to-date.
[  +22 ms] ClCompile:
[        ]   C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.34.31933\bin\HostX64\x64\CL.exe /c /I"C:\Users\allenm\Desktop\work\artc\artc-frontend\artc_frontend\windows" /I"C:\Users\allenm\Desktop\work\artc\artc-frontend\artc_frontend\windows\flutter\ephemeral" /I"C:\Users\allenm\Desktop\work\artc\artc-frontend\artc_frontend\windows\flutter\ephemeral\cpp_client_wrapper\include" /Zi /nologo /W4 /WX /diagnostics:column /Od /Ob0 /D _UNICODE /D UNICODE /D WIN32 /D _WINDOWS /D _HAS_EXCEPTIONS=0 /D _DEBUG /D "FLUTTER_VERSION=\"1.0.0+1\"" /D FLUTTER_VERSION_MAJOR=1 /D FLUTTER_VERSION_MINOR=0 /D FLUTTER_VERSION_PATCH=0 /D FLUTTER_VERSION_BUILD=1 /D NOMINMAX /D UNICODE /D _UNICODE /D "CMAKE_INTDIR=\"Debug\"" /Gm- /EHsc /RTC1 /MDd /GS /fp:precise /Zc:wchar_t /Zc:forScope /Zc:inline /GR /std:c++17 /Fo"artc_frontend.dir\Debug\\" /Fd"artc_frontend.dir\Debug\vc143.pdb" /external:W4 /Gd /TP /wd4100 /errorReport:queue "C:\Users\allenm\Desktop\work\artc\artc-frontend\artc_frontend\windows\flutter\generated_plugin_registrant.cc" "C:\Users\allenm\Desktop\work\artc\artc-frontend\artc_frontend\windows\runner\flutter_window.cpp"
[ +112 ms]   generated_plugin_registrant.cc
[ +744 ms]   flutter_window.cpp
[+1189 ms]   Generating Code...
[ +164 ms] ResourceCompile:
[        ]   All outputs are up-to-date.
[  +58 ms] Link:
[        ]   C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.34.31933\bin\HostX64\x64\link.exe /ERRORREPORT:QUEUE /OUT:"C:\Users\allenm\Desktop\work\artc\artc-frontend\artc_frontend\build\windows\runner\Debug\artc_frontend.exe" /INCREMENTAL /ILK:"artc_frontend.dir\Debug\artc_frontend.ilk" /NOLOGO ..\flutter\Debug\flutter_wrapper_app.lib "C:\Users\allenm\Desktop\work\artc\artc-frontend\artc_frontend\windows\flutter\ephemeral\flutter_windows.dll.lib" kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib /MANIFEST /MANIFESTUAC:"level='asInvoker' uiAccess='false'" /manifest:embed /manifestinput:"C:\Users\allenm\Desktop\work\artc\artc-frontend\artc_frontend\windows\runner\runner.exe.manifest" /DEBUG /PDB:"C:/Users/allenm/Desktop/work/artc/artc-frontend/artc_frontend/build/windows/runner/Debug/artc_frontend.pdb" /SUBSYSTEM:WINDOWS /TLBID:1 /DYNAMICBASE /NXCOMPAT /IMPLIB:"C:/Users/allenm/Desktop/work/artc/artc-frontend/artc_frontend/build/windows/runner/Debug/artc_frontend.lib" /MACHINE:X64  /machine:x64 artc_frontend.dir\Debug\Runner.res
[        ]   artc_frontend.dir\Debug\flutter_window.obj
[        ]   artc_frontend.dir\Debug\main.obj
[        ]   artc_frontend.dir\Debug\utils.obj
[        ]   artc_frontend.dir\Debug\win32_window.obj
[        ]   artc_frontend.dir\Debug\generated_plugin_registrant.obj
[ +359 ms]   artc_frontend.vcxproj -> C:\Users\allenm\Desktop\work\artc\artc-frontend\artc_frontend\build\windows\runner\Debug\artc_frontend.exe
[  +11 ms] FinalizeBuildStatus:
[        ]   Deleting file "artc_frontend.dir\Debug\artc_frontend.tlog\unsuccessfulbuild".
[        ]   Touching "artc_frontend.dir\Debug\artc_frontend.tlog\artc_frontend.lastbuildstate".
[   +1 ms] Done Building Project "C:\Users\allenm\Desktop\work\artc\artc-frontend\artc_frontend\build\windows\runner\artc_frontend.vcxproj" (default targets).
[  +30 ms] Project "C:\Users\allenm\Desktop\work\artc\artc-frontend\artc_frontend\build\windows\ALL_BUILD.vcxproj" (3) is building "C:\Users\allenm\Desktop\work\artc\artc-frontend\artc_frontend\build\windows\flutter\flutter_wrapper_plugin.vcxproj" (7) on node 1 (default targets).
[        ] InitializeBuildStatus:
[        ]   Creating "flutter_wrapper_plugin.dir\Debug\flutter_.D7ACF3D2.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[   +2 ms] CustomBuild:
[        ]   All outputs are up-to-date.
[  +14 ms] ClCompile:
[        ]   All outputs are up-to-date.
[   +5 ms] Lib:
[        ]   All outputs are up-to-date.
[        ]   flutter_wrapper_plugin.vcxproj -> C:\Users\allenm\Desktop\work\artc\artc-frontend\artc_frontend\build\windows\flutter\Debug\flutter_wrapper_plugin.lib
[   +7 ms] FinalizeBuildStatus:
[        ]   Deleting file "flutter_wrapper_plugin.dir\Debug\flutter_.D7ACF3D2.tlog\unsuccessfulbuild".
[        ]   Touching "flutter_wrapper_plugin.dir\Debug\flutter_.D7ACF3D2.tlog\flutter_wrapper_plugin.lastbuildstate".
[        ] Done Building Project "C:\Users\allenm\Desktop\work\artc\artc-frontend\artc_frontend\build\windows\flutter\flutter_wrapper_plugin.vcxproj" (default targets).
[   +8 ms] InitializeBuildStatus:
[        ]   Creating "x64\Debug\ALL_BUILD\ALL_BUILD.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[   +3 ms] CustomBuild:
[        ]   All outputs are up-to-date.
[  +13 ms] FinalizeBuildStatus:
[        ]   Deleting file "x64\Debug\ALL_BUILD\ALL_BUILD.tlog\unsuccessfulbuild".
[        ]   Touching "x64\Debug\ALL_BUILD\ALL_BUILD.tlog\ALL_BUILD.lastbuildstate".
[   +1 ms] Done Building Project "C:\Users\allenm\Desktop\work\artc\artc-frontend\artc_frontend\build\windows\ALL_BUILD.vcxproj" (default targets).
[   +9 ms] InitializeBuildStatus:
[        ]   Creating "x64\Debug\INSTALL\INSTALL.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[  +89 ms] PostBuildEvent:
[        ]   setlocal
[        ]   "C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe" -DBUILD_TYPE=Debug -P cmake_install.cmake
[        ]   if %errorlevel% neq 0 goto :cmEnd
[        ]   :cmEnd
[        ]   endlocal & call :cmErrorLevel %errorlevel% & goto :cmDone
[        ]   :cmErrorLevel
[        ]   exit /b %1
[        ]   :cmDone
[        ]   if %errorlevel% neq 0 goto :VCEnd
[        ]   :VCEnd
[ +116 ms]   -- Install configuration: "Debug"
[   +2 ms]   -- Up-to-date: C:/Users/allenm/Desktop/work/artc/artc-frontend/artc_frontend/build/windows/runner/Debug/data/icudtl.dat
[        ]   -- Up-to-date: C:/Users/allenm/Desktop/work/artc/artc-frontend/artc_frontend/build/windows/runner/Debug/flutter_windows.dll
[   +9 ms]   -- Installing: C:/Users/allenm/Desktop/work/artc/artc-frontend/artc_frontend/build/windows/runner/Debug/data/flutter_assets
[   +1 ms]   -- Installing: C:/Users/allenm/Desktop/work/artc/artc-frontend/artc_frontend/build/windows/runner/Debug/data/flutter_assets/AssetManifest.json
[   +2 ms]   -- Installing: C:/Users/allenm/Desktop/work/artc/artc-frontend/artc_frontend/build/windows/runner/Debug/data/flutter_assets/FontManifest.json
[   +2 ms]   -- Installing: C:/Users/allenm/Desktop/work/artc/artc-frontend/artc_frontend/build/windows/runner/Debug/data/flutter_assets/fonts
[   +1 ms]   -- Installing: C:/Users/allenm/Desktop/work/artc/artc-frontend/artc_frontend/build/windows/runner/Debug/data/flutter_assets/fonts/MaterialIcons-Regular.otf
[   +9 ms]   -- Installing: C:/Users/allenm/Desktop/work/artc/artc-frontend/artc_frontend/build/windows/runner/Debug/data/flutter_assets/kernel_blob.bin
[ +292 ms]   -- Installing: C:/Users/allenm/Desktop/work/artc/artc-frontend/artc_frontend/build/windows/runner/Debug/data/flutter_assets/NOTICES.Z
[   +2 ms]   -- Installing: C:/Users/allenm/Desktop/work/artc/artc-frontend/artc_frontend/build/windows/runner/Debug/data/flutter_assets/packages
[   +1 ms]   -- Installing: C:/Users/allenm/Desktop/work/artc/artc-frontend/artc_frontend/build/windows/runner/Debug/data/flutter_assets/packages/cupertino_icons
[   +1 ms]   -- Installing: C:/Users/allenm/Desktop/work/artc/artc-frontend/artc_frontend/build/windows/runner/Debug/data/flutter_assets/packages/cupertino_icons/assets
[   +1 ms]   -- Installing: C:/Users/allenm/Desktop/work/artc/artc-frontend/artc_frontend/build/windows/runner/Debug/data/flutter_assets/packages/cupertino_icons/assets/CupertinoIcons.ttf
[   +5 ms]   -- Installing: C:/Users/allenm/Desktop/work/artc/artc-frontend/artc_frontend/build/windows/runner/Debug/data/flutter_assets/packages/wakelock_web
[   +1 ms]   -- Installing: C:/Users/allenm/Desktop/work/artc/artc-frontend/artc_frontend/build/windows/runner/Debug/data/flutter_assets/packages/wakelock_web/assets
[   +1 ms]   -- Installing: C:/Users/allenm/Desktop/work/artc/artc-frontend/artc_frontend/build/windows/runner/Debug/data/flutter_assets/packages/wakelock_web/assets/no_sleep.js
[   +3 ms]   -- Installing: C:/Users/allenm/Desktop/work/artc/artc-frontend/artc_frontend/build/windows/runner/Debug/data/flutter_assets/shaders
[   +1 ms]   -- Installing: C:/Users/allenm/Desktop/work/artc/artc-frontend/artc_frontend/build/windows/runner/Debug/data/flutter_assets/shaders/ink_sparkle.frag
[  +21 ms] FinalizeBuildStatus:
[        ]   Deleting file "x64\Debug\INSTALL\INSTALL.tlog\unsuccessfulbuild".
[        ]   Touching "x64\Debug\INSTALL\INSTALL.tlog\INSTALL.lastbuildstate".
[   +1 ms] Done Building Project "C:\Users\allenm\Desktop\work\artc\artc-frontend\artc_frontend\build\windows\INSTALL.vcxproj" (default targets).
[   +8 ms] Build succeeded.
[        ]     0 Warning(s)
[        ]     0 Error(s)
[        ] Time Elapsed 00:00:13.83
[  +49 ms] Building Windows application... (completed in 14.7s)
[+1347 ms] Observatory URL on device: http://127.0.0.1:59133/B-C_v1V7DAQ=/
[   +5 ms] Caching compiled dill
[  +54 ms] Connecting to service protocol: http://127.0.0.1:59133/B-C_v1V7DAQ=/
[ +231 ms] Launching a Dart Developer Service (DDS) instance at http://127.0.0.1:0, connecting to VM service at http://127.0.0.1:59133/B-C_v1V7DAQ=/.
[ +133 ms] DDS is listening at http://127.0.0.1:59139/8f3SMtTpzUA=/.
[  +98 ms] Successfully connected to service protocol: http://127.0.0.1:59133/B-C_v1V7DAQ=/
[ +313 ms] DevFS: Creating new filesystem on the device (null)
[  +31 ms] DevFS: Created new filesystem on the device (file:///C:/Users/allenm/AppData/Local/Temp/artc_frontend1f6e4560/artc_frontend/)
[   +3 ms] Updating assets
[ +300 ms] Syncing files to device Windows...
[   +2 ms] Compiling dart to kernel with 0 updated files
[        ] Processing bundle.
[   +2 ms] <- recompile package:artc_frontend/main.dart 8a6f5c9b-8c38-4176-95aa-6d6f726772ab
[        ] <- 8a6f5c9b-8c38-4176-95aa-6d6f726772ab
[   +3 ms] Bundle processing done.
[        ] flutter: ____DBG: in initSate
[        ] flutter: ____DBG: starting initializePlayer
[        ] flutter: ____DBG: VideoPlayerController Made
[        ] flutter: ____DBG: CONTROLLER SHOULD BE INITIALIZED NOW..
[        ] flutter: ____DBG:  initState DONE
[ +262 ms] [ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: UnimplementedError: init() has not been implemented.
[        ] #0      VideoPlayerPlatform.init (package:video_player_platform_interface/video_player_platform_interface.dart:43:5)
[        ] #1      _videoPlayerPlatform (package:video_player/video_player.dart:28:21)
[        ] #2      VideoPlayerController.initialize (package:video_player/video_player.dart:365:25)
[        ] #3      _ContentScreenState.initializePlayer (package:artc_frontend/pages/home/content_screen.dart:59:34)
[        ] #4      _ContentScreenState.initState (package:artc_frontend/pages/home/content_screen.dart:25:5)
[        ] #5      StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:5015:57)
[        ] #6      ComponentElement.mount (package:flutter/src/widgets/framework.dart:4853:5)
[        ] #7      Element.inflateWidget (package:flutter/src/widgets/framework.dart:3863:16)
[        ] #8      Element.updateChild (package:flutter/src/widgets/framework.dart:3592:18)
[        ] #9      ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:4904:16)
[        ] #10     StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5050:11)
[        ] #11     Element.rebuild (package:flutter/src/widgets/framework.dart:4604:5)
[        ] #12     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:4859:5)
[        ] #13     StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:5041:11)
[        ] #14     ComponentElement.mount (package:flutter/src/widgets/framework.dart:4853:5)
[        ] #15     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3863:16)
[        ] #16     Element.updateChild (package:flutter/src/widgets/framework.dart:3592:18)
[        ] #17     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:6300:14)
[        ] #18     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3863:16)
[        ] #19     Element.updateChild (package:flutter/src/widgets/framework.dart:3592:18)
[        ] #20     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:6300:14)
[        ] #21     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3863:16)
[        ] #22     Element.updateChild (package:flutter/src/widgets/framework.dart:3592:18)
[        ] #23     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:4904:16)
[        ] #24     StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5050:11)
[        ] #25     Element.rebuild (package:flutter/src/widgets/framework.dart:4604:5)
[        ] #26     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:4859:5)
[        ] #27     StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:5041:11)
[        ] #28     ComponentElement.mount (package:flutter/src/widgets/framework.dart:4853:5)
[        ] #29     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3863:16)
[        ] #30     Element.updateChild (package:flutter/src/widgets/framework.dart:3592:18)
[        ] #31     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:4904:16)
[        ] #32     Element.rebuild (package:flutter/src/widgets/framework.dart:4604:5)
[        ] #33     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:4859:5)
[        ] #34     ComponentElement.mount (package:flutter/src/widgets/framework.dart:4853:5)
[        ] #35     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3863:16)
[        ] #36     Element.updateChild (package:flutter/src/widgets/framework.dart:3592:18)
[        ] #37     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:4904:16)
[        ] #38     Element.rebuild (package:flutter/src/widgets/framework.dart:4604:5)
[        ] #39     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:4859:5)
[        ] #40     ComponentElement.mount (package:flutter/src/widgets/framework.dart:4853:5)
[        ] #41     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3863:16)
[   +1 ms] #42     Element.updateChild (package:flutter/src/widgets/framework.dart:3592:18)
[        ] #43     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:4904:16)
[        ] #44     StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5050:11)
[        ] #45     Element.rebuild (package:flutter/src/widgets/framework.dart:4604:5)
[        ] #46     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:4859:5)
[        ] #47     StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:5041:11)
[        ] #48     ComponentElement.mount (package:flutter/src/widgets/framework.dart:4853:5)
[        ] #49     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3863:16)
[        ] #50     Element.updateChild (package:flutter/src/widgets/framework.dart:3592:18)
[        ] #51     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:4904:16)
[        ] #52     Element.rebuild (package:flutter/src/widgets/framework.dart:4604:5)
[        ] #53     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:4859:5)
[        ] #54     ComponentElement.mount (package:flutter/src/widgets/framework.dart:4853:5)
[        ] #55     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3863:16)
[        ] #56     Element.updateChild (package:flutter/src/widgets/framework.dart:3592:18)
[        ] #57     SliverMultiBoxAdaptorElement.updateChild (package:flutter/src/widgets/sliver.dart:1405:37)
[        ] #58     SliverMultiBoxAdaptorElement.createChild.<anonymous closure> (package:flutter/src/widgets/sliver.dart:1390:20)
[        ] #59     BuildOwner.buildScope (package:flutter/src/widgets/framework.dart:2605:19)
[        ] #60     SliverMultiBoxAdaptorElement.createChild (package:flutter/src/widgets/sliver.dart:1382:12)
[        ] #61     RenderSliverMultiBoxAdaptor._createOrObtainChild.<anonymous closure> (package:flutter/src/rendering/sliver_multi_box_adaptor.dart:353:23)
[        ] #62     RenderObject.invokeLayoutCallback.<anonymous closure> (package:flutter/src/rendering/object.dart:2246:59)
[        ] #63     PipelineOwner._enableMutationsToDirtySubtrees (package:flutter/src/rendering/object.dart:1035:15)
[        ] #64     RenderObject.invokeLayoutCallback (package:flutter/src/rendering/object.dart:2246:14)
[        ] #65     RenderSliverMultiBoxAdaptor._createOrObtainChild (package:flutter/src/rendering/sliver_multi_box_adaptor.dart:342:5)
[        ] #66     RenderSliverMultiBoxAdaptor.addInitialChild (package:flutter/src/rendering/sliver_multi_box_adaptor.dart:428:5)
[        ] #67     RenderSliverFixedExtentBoxAdaptor.performLayout (package:flutter/src/rendering/sliver_fixed_extent_list.dart:205:12)
[        ] #68     RenderObject.layout (package:flutter/src/rendering/object.dart:2135:7)
[        ] #69     RenderSliverEdgeInsetsPadding.performLayout (package:flutter/src/rendering/sliver_padding.dart:149:12)
[        ] #70     _RenderSliverFractionalPadding.performLayout (package:flutter/src/widgets/sliver_fill.dart:166:11)
[        ] #71     RenderObject.layout (package:flutter/src/rendering/object.dart:2135:7)
[        ] #72     RenderViewportBase.layoutChildSequence (package:flutter/src/rendering/viewport.dart:516:13)
[        ] #73     RenderViewport._attemptLayout (package:flutter/src/rendering/viewport.dart:1600:12)
[        ] #74     RenderViewport.performLayout (package:flutter/src/rendering/viewport.dart:1507:20)
[        ] #75     RenderObject.layout (package:flutter/src/rendering/object.dart:2135:7)
[        ] #76     RenderBox.layout (package:flutter/src/rendering/box.dart:2418:11)
[        ] #77     RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:120:14)
[        ] #78     RenderObject.layout (package:flutter/src/rendering/object.dart:2135:7)
[        ] #79     RenderBox.layout (package:flutter/src/rendering/box.dart:2418:11)
[        ] #80     RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:120:14)
[        ] #81     RenderObject.layout (package:flutter/src/rendering/object.dart:2135:7)
[        ] #82     RenderBox.layout (package:flutter/src/rendering/box.dart:2418:11)
[        ] #83     RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:120:14)
[        ] #84     RenderObject.layout (package:flutter/src/rendering/object.dart:2135:7)
[        ] #85     RenderBox.layout (package:flutter/src/rendering/box.dart:2418:11)
[        ] #86     RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:120:14)
[        ] #87     RenderObject.layout (package:flutter/src/rendering/object.dart:2135:7)
[        ] #88     RenderBox.layout (package:flutter/src/rendering/box.dart:2418:11)
[        ] #89     RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:120:14)
[        ] #90     RenderObject.layout (package:flutter/src/rendering/object.dart:2135:7)
[        ] #91     RenderBox.layout (package:flutter/src/rendering/box.dart:2418:11)
[        ] #92     RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:120:14)
[        ] #93     RenderObject.layout (package:flutter/src/rendering/object.dart:2135:7)
[        ] #94     RenderBox.layout (package:flutter/src/rendering/box.dart:2418:11)
[        ] #95     MultiChildLayoutDelegate.layoutChild (package:flutter/src/rendering/custom_layout.dart:171:12)
[        ] #96     _ScaffoldLayout.performLayout (package:flutter/src/material/scaffold.dart:1055:7)
[        ] #97     MultiChildLayoutDelegate._callPerformLayout (package:flutter/src/rendering/custom_layout.dart:240:7)
[        ] #98     RenderCustomMultiChildLayoutBox.performLayout (package:flutter/src/rendering/custom_layout.dart:410:14)
[        ] #99     RenderObject.layout (package:flutter/src/rendering/object.dart:2135:7)
[        ] #100    RenderBox.layout (package:flutter/src/rendering/box.dart:2418:11)
[        ] #101    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:120:14)
[        ] #102    RenderObject.layout (package:flutter/src/rendering/object.dart:2135:7)
[        ] #103    RenderBox.layout (package:flutter/src/rendering/box.dart:2418:11)
[        ] #104    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:120:14)
[        ] #105    _RenderCustomClip.performLayout (package:flutter/src/rendering/proxy_box.dart:1462:11)
[        ] #106    RenderObject.layout (package:flutter/src/rendering/object.dart:2135:7)
[        ] #107    RenderBox.layout (package:flutter/src/rendering/box.dart:2418:11)
[        ] #108    RenderPositionedBox.performLayout (package:flutter/src/rendering/shifted_box.dart:451:14)
[        ] #109    RenderObject.layout (package:flutter/src/rendering/object.dart:2135:7)
[        ] #110    RenderBox.layout (package:flutter/src/rendering/box.dart:2418:11)
[        ] #111    MultiChildLayoutDelegate.layoutChild (package:flutter/src/rendering/custom_layout.dart:171:12)
[   +1 ms] #112    _ScaffoldLayout.performLayout (package:flutter/src/material/scaffold.dart:1055:7)
[        ] #113    MultiChildLayoutDelegate._callPerformLayout (package:flutter/src/rendering/custom_layout.dart:240:7)
[        ] #114    RenderCustomMultiChildLayoutBox.performLayout (package:flutter/src/rendering/custom_layout.dart:410:14)
[        ] #115    RenderObject.layout (package:flutter/src/rendering/object.dart:2135:7)
[        ] #116    RenderBox.layout (package:flutter/src/rendering/box.dart:2418:11)
[        ] #117    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:120:14)
[        ] #118    RenderObject.layout (package:flutter/src/rendering/object.dart:2135:7)
[        ] #119    RenderBox.layout (package:flutter/src/rendering/box.dart:2418:11)
[        ] #120    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:120:14)
[        ] #121    _RenderCustomClip.performLayout (package:flutter/src/rendering/proxy_box.dart:1462:11)
[        ] #122    RenderObject.layout (package:flutter/src/rendering/object.dart:2135:7)
[        ] #123    RenderBox.layout (package:flutter/src/rendering/box.dart:2418:11)
[        ] #124    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:120:14)
[        ] #125    RenderObject.layout (package:flutter/src/rendering/object.dart:2135:7)
[        ] #126    RenderBox.layout (package:flutter/src/rendering/box.dart:2418:11)
[        ] #127    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:120:14)
[        ] #128    RenderObject.layout (package:flutter/src/rendering/object.dart:2135:7)
[        ] #129    RenderBox.layout (package:flutter/src/rendering/box.dart:2418:11)
[        ] #130    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:120:14)
[        ] #131    RenderObject.layout (package:flutter/src/rendering/object.dart:2135:7)
[        ] #132    RenderBox.layout (package:flutter/src/rendering/box.dart:2418:11)
[        ] #133    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:120:14)
[        ] #134    RenderObject.layout (package:flutter/src/rendering/object.dart:2135:7)
[        ] #135    RenderBox.layout (package:flutter/src/rendering/box.dart:2418:11)
[        ] #136    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:120:14)
[        ] #137    RenderObject.layout (package:flutter/src/rendering/object.dart:2135:7)
[        ] #138    RenderBox.layout (package:flutter/src/rendering/box.dart:2418:11)
[        ] #139    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:120:14)
[        ] #140    RenderObject.layout (package:flutter/src/rendering/object.dart:2135:7)
[        ] #141    RenderBox.layout (package:flutter/src/rendering/box.dart:2418:11)
[        ] #142    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:120:14)
[        ] #143    RenderObject.layout (package:flutter/src/rendering/object.dart:2135:7)
[        ] #144    RenderBox.layout (package:flutter/src/rendering/box.dart:2418:11)
[        ] #145    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:120:14)
[        ] #146    RenderObject.layout (package:flutter/src/rendering/object.dart:2135:7)
[        ] #147    RenderBox.layout (package:flutter/src/rendering/box.dart:2418:11)
[        ] #148    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:120:14)
[        ] #149    RenderObject.layout (package:flutter/src/rendering/object.dart:2135:7)
[        ] #150    RenderBox.layout (package:flutter/src/rendering/box.dart:2418:11)
[        ] #151    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:120:14)
[        ] #152    RenderObject.layout (package:flutter/src/rendering/object.dart:2135:7)
[        ] #153    RenderBox.layout (package:flutter/src/rendering/box.dart:2418:11)
[        ] #154    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:120:14)
[        ] #155    RenderObject.layout (package:flutter/src/rendering/object.dart:2135:7)
[        ] #156    RenderBox.layout (package:flutter/src/rendering/box.dart:2418:11)
[        ] #157    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:120:14)
[        ] #158    RenderObject.layout (package:flutter/src/rendering/object.dart:2135:7)
[        ] #159    RenderBox.layout (package:flutter/src/rendering/box.dart:2418:11)
[        ] #160    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:120:14)
[        ] #161    RenderObject.layout (package:flutter/src/rendering/object.dart:2135:7)
[        ] #162    RenderBox.layout (package:flutter/src/rendering/box.dart:2418:11)
[        ] #163    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:120:14)
[        ] #164    RenderObject.layout (package:flutter/src/rendering/object.dart:2135:7)
[        ] #165    RenderBox.layout (package:flutter/src/rendering/box.dart:2418:11)
[        ] #166    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:120:14)
[        ] #167    RenderObject.layout (package:flutter/src/rendering/object.dart:2135:7)
[        ] #168    RenderBox.layout (package:flutter/src/rendering/box.dart:2418:11)
[        ] #169    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:120:14)
[        ] #170    RenderObject.layout (package:flutter/src/rendering/object.dart:2135:7)
[        ] #171    RenderBox.layout (package:flutter/src/rendering/box.dart:2418:11)
[        ] #172    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:120:14)
[        ] #173    RenderOffstage.performLayout (package:flutter/src/rendering/proxy_box.dart:3737:13)
[        ] #174    RenderObject.layout (package:flutter/src/rendering/object.dart:2135:7)
[        ] #175    RenderBox.layout (package:flutter/src/rendering/box.dart:2418:11)
[        ] #176    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:120:14)
[        ] #177    RenderObject.layout (package:flutter/src/rendering/object.dart:2135:7)
[        ] #178    RenderBox.layout (package:flutter/src/rendering/box.dart:2418:11)
[        ] #179    _RenderTheatre.performLayout (package:flutter/src/widgets/overlay.dart:804:15)
[        ] #180    RenderObject.layout (package:flutter/src/rendering/object.dart:2135:7)
[        ] #181    RenderBox.layout (package:flutter/src/rendering/box.dart:2418:11)
[        ] #182    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:120:14)
[        ] #183    RenderObject.layout (package:flutter/src/rendering/object.dart:2135:7)
[        ] #184    RenderBox.layout (package:flutter/src/rendering/box.dart:2418:11)
[        ] #185    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:120:14)
[        ] #186    RenderObject.layout (package:flutter/src/rendering/object.dart:2135:7)
[        ] #187    RenderBox.layout (package:flutter/src/rendering/box.dart:2418:11)
[        ] #188    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:120:14)
[        ] #189    RenderObject.layout (package:flutter/src/rendering/object.dart:2135:7)
[        ] #190    RenderBox.layout (package:flutter/src/rendering/box.dart:2418:11)
[        ] #191    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:120:14)
[        ] #192    RenderCustomPaint.performLayout (package:flutter/src/rendering/custom_paint.dart:552:11)
[        ] #193    RenderObject.layout (package:flutter/src/rendering/object.dart:2135:7)
[        ] #194    RenderBox.layout (package:flutter/src/rendering/box.dart:2418:11)
[        ] #195    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:120:14)
[        ] #196    RenderObject.layout (package:flutter/src/rendering/object.dart:2135:7)
[        ] #197    RenderBox.layout (package:flutter/src/rendering/box.dart:2418:11)
[        ] #198    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:120:14)
[        ] #199    RenderObject.layout (package:flutter/src/rendering/object.dart:2135:7)
[        ] #200    RenderBox.layout (package:flutter/src/rendering/box.dart:2418:11)
[        ] #201    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:120:14)
[        ] #202    RenderObject.layout (package:flutter/src/rendering/object.dart:2135:7)
[        ] #203    RenderBox.layout (package:flutter/src/rendering/box.dart:2418:11)
[        ] #204    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:120:14)
[        ] #205    RenderObject.layout (package:flutter/src/rendering/object.dart:2135:7)
[        ] #206    RenderBox.layout (package:flutter/src/rendering/box.dart:2418:11)
[        ] #207    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:120:14)
[        ] #208    RenderObject.layout (package:flutter/src/rendering/object.dart:2135:7)
[        ] #209    RenderBox.layout (package:flutter/src/rendering/box.dart:2418:11)
[        ] #210    RenderView.performLayout (package:flutter/src/rendering/view.dart:170:14)
[        ] #211    RenderObject._layoutWithoutResize (package:flutter/src/rendering/object.dart:1973:7)
[        ] #212    PipelineOwner.flushLayout (package:flutter/src/rendering/object.dart:999:18)
[        ] #213    RendererBinding.drawFrame (package:flutter/src/rendering/binding.dart:513:19)
[        ] #214    WidgetsBinding.drawFrame (package:flutter/src/widgets/binding.dart:884:13)
[        ] #215    RendererBinding._handlePersistentFrameCallback (package:flutter/src/rendering/binding.dart:378:5)
[        ] #216    SchedulerBinding._invokeFrameCallback (package:flutter/src/scheduler/binding.dart:1175:15)
[        ] #217    SchedulerBinding.handleDrawFrame (package:flutter/src/scheduler/binding.dart:1104:9)
[        ] #218    SchedulerBinding.scheduleWarmUpFrame.<anonymous closure> (package:flutter/src/scheduler/binding.dart:881:7)
[        ] #219    Timer._createTimer.<anonymous closure> (dart:async-patch/timer_patch.dart:18:15)
[        ] #220    _Timer._runTimers (dart:isolate-patch/timer_impl.dart:398:19)
[        ] #221    _Timer._handleMessage (dart:isolate-patch/timer_impl.dart:429:5)
[        ] #222    _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:192:12)
[   +1 ms] Updating files.
[        ] DevFS: Sync finished
[   +2 ms] Syncing files to device Windows... (completed in 327ms)
[        ] Synced 0.0MB.
[   +2 ms] <- accept
[   +3 ms] Connected to _flutterView/0x17fad2c4270.
[   +3 ms] Flutter run key commands.
[   +1 ms] r Hot reload. 
[        ] R Hot restart.
[        ] h List all available interactive commands.
[        ] d Detach (terminate "flutter run" but leave application running).
[        ] c Clear the screen
[        ] q Quit (terminate the application on the device).
[        ]  Running with sound null safety 
[   +1 ms] An Observatory debugger and profiler on Windows is available at: http://127.0.0.1:59139/8f3SMtTpzUA=/
[  +11 ms] [ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: UnimplementedError: init() has not been implemented.
[        ] #0      VideoPlayerPlatform.init (package:video_player_platform_interface/video_player_platform_interface.dart:43:5)
[        ] #1      _videoPlayerPlatform (package:video_player/video_player.dart:28:21)
[        ] #2      VideoPlayerController.initialize (package:video_player/video_player.dart:365:25)
[        ] #3      ChewieController._initialize (package:chewie/src/chewie_player.dart:552:35)
[        ] <asynchronous suspension>
[  +88 ms] The Flutter DevTools debugger and profiler on Windows is available at: http://127.0.0.1:59142?uri=http://127.0.0.1:59139/8f3SMtTpzUA=/

```

</details>

**Comment by stuartmorgan** [comment]
`video_player` doesn't support Windows, so this is expected behavior. Closing as a duplicate of https://github.com/flutter/flutter/issues/37673.
